frontend/awx/resources/inventories/hooks/useDeleteInventories.tsx

```diff
@@ -4,15 +4,23 @@ import { compareStrings, useBulkConfirmation } from '../../../../../framework';
 import { useNameColumn } from '../../../../common/columns';
 import { getItemKey, requestDelete } from '../../../../common/crud/Data';
 import { Inventory } from '../../../interfaces/Inventory';
-import { useInventoriesColumns } from '../Inventories';
+import { useInventoriesColumns } from './useInventoriesColumns';
 
 export function useDeleteInventories(onComplete: (inventories: Inventory[]) => void) {
   const { t } = useTranslation();
   const confirmationColumns = useInventoriesColumns({ disableLinks: true, disableSort: true });
   const deleteActionNameColumn = useNameColumn({ disableLinks: true, disableSort: true });
   const actionColumns = useMemo(() => [deleteActionNameColumn], [deleteActionNameColumn]);
   const bulkAction = useBulkConfirmation<Inventory>();
+
+  const cannotDeleteInventory = (inventory: Inventory) => {
+    return inventory?.summary_fields?.user_capabilities?.delete
+      ? undefined
+      : t('The inventory cannot be deleted due to insufficient permissions.');
+  };
+
   const deleteInventories = (inventories: Inventory[]) => {
+    const undeletableInventories = inventories.filter(cannotDeleteInventory);
     bulkAction({
       title:
         inventories.length === 1
```

Comment:
We need to update the translation for the confirmation message in case of a singular inventory. It currently displays `Yes, I confirm that I want to delete these 1 inventories.`.